### ISSUE TITLE
Fix use of ln on OS X [phab:D895]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1204]
_Created by **zarko** at 2016-05-16 23:48:24_: 

This change unbreaks builds after clean (and possibly some
other scenarios, but I was reliably able to get it to trip
using `bazel clean`):

```
$ bazel clean --expunge
$ bazel build //kythe/go/storage/leveldb/...
INFO: Found 2 targets...
...
INFO: Elapsed time: 67.263s, Critical Path: 29.49s
$ bazel clean
INFO: Starting clean (this may take a while). Consider using --expunge_async if the clean takes more than several minutes.
$ bazel build //kythe/go/storage/leveldb/...
INFO: Found 2 targets...
ERROR: /private/var/tmp/.../2be6994bf77f75bac8f1433eb6e9dc33/external/go_levigo/BUILD:9:1: error executing shell command: 'set -e
export GOROOT=$PWD/external/local_goroot
export CC=$PWD/external/local_config_cc/clang
export CGO_CFLAGS="-DLEVELDB_PLATFORM_POSIX -DOS_MACOSX -isystem $PWD/third_party/leveldb/include -isys...' failed: bash failed: error executing command /bin/bash -c ... (remaining 1 argument(s) skipped): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
ln: /var/folders/ft/bv0xnfpx2bqbqc1zp24gc4n8001p27/T//gopath/src/github.com/jmhodges/levigo/go_levigo: File exists
INFO: Elapsed time: 24.082s, Critical Path: 20.79s
```

**Comment by kythe** [comment]
_**schroederc** accepted this revision at 2016-05-17 04:01:26_

**Comment by kythe** [comment]
_**zarko** closed via commit 432309a547339c86f43dda8058fb450608c57f89 at 2016-05-17 17:48:48_
